org-code-javabuilder/lib/src/main/java/org/code/javabuilder/CodeBuilder.java

```diff
@@ -85,4 +86,19 @@ public void close() throws InternalFacingException {
       }
     }
   }
+
+  /** Save any non-source code files to storage */
+  private void saveProjectAssets() throws UserFacingException {
+    // Save all text files to current folder.
+    List<TextProjectFile> textProjectFiles = this.userProjectFiles.getTextFiles();
+    for (TextProjectFile projectFile : textProjectFiles) {
+      String filePath = projectFile.getFileName();
+      try (PrintWriter out = new PrintWriter(filePath)) {
```

Comment:
Not sure if it's easier or not, but `Files.writeString` might be an option here.

Reply:
updated!